meta/saisanitycheck.c

```diff
@@ -567,6 +567,8 @@ void check_attr_default_required(
             break;
 
         case SAI_ATTR_VALUE_TYPE_UINT8_LIST:
+        case SAI_ATTR_VALUE_TYPE_STRING:
+        case SAI_ATTR_VALUE_TYPE_POINTER:
```

Comment:
those are not list types, please move them to separate case statement and allow default value type as CONST or NONE only